README.md

```diff
@@ -1,106 +1,122 @@
+Kindly go through [CONTRIBUTING.md](https://github.com/krishnaacharyaa/wanderlust/blob/main/.github/CONTRIBUTING.md) to understand everything from setup to contributing guidelines.
 
-   ```bash
-   mongoimport --db wanderlust --collection posts --file ./data/sample_posts.json --jsonArray
-   ```
+If you would like to contribute to the project, please follow our contribution guidelines.
 
-6. **Configure Environment Variables**
+<hr>
+<!-- Open Source Programs -->
+  <div>
+    <h2><img src="https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/master/Emojis/Hand%20gestures/Flexed%20Biceps.png?raw=true" width="35" height="35" > Open Source Programs</h2>
+  </div>
 
-   ```bash
-   cp .env.sample .env
-   ```
+   <div>
+  <h3>This project is part of <img src="https://github.com/Meetjain1/wanderlust/assets/133582566/c0def4be-3f50-4b6a-8781-91856a8a6822" width="30" height="20">GirlScript Summer of Code. We warmly welcome contributions from the community to help elevate Wanderlust.
+  </h3>
+  </div>
 
-7. **Start the Backend Server**
+<hr>
 
-   ```bash
-   npm start
-   ```
+<!-- Code of conduct -->
+<div>
+<h2><img src = "https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Hand%20gestures/Handshake.png" width="35" height="35"> Code of Conduct</h2>
+</div>
 
-   > You should see the following on your terminal output on successful setup.
-   >
-   > ```bash
-   > [BACKEND] Server is running on port 5000
-   > [BACKEND] Database connected: mongodb://127.0.0.1/wanderlust
-   > ```
+Please note that this project is released with a [Contributor Code of Conduct](.github/CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
 
-### Setting up the Frontend
+<hr>
 
-1. **Open a New Terminal**
+<!-- License -->
+<div>
+<h2><img src = "https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Objects/Page%20with%20Curl.png" width="35" height="35"> License</h2>
+</div>
 
-   ```bash
-   cd frontend
-   ```
+This project is licensed under the [MIT License](./LICENSE).
 
-2. **Install Dependencies**
+<hr>
 
-   ```bash
-   npm i
-   ```
+<div>
+  <h2><img src="https://github.com/Meetjain1/wanderlust/assets/133582566/af38ec84-7387-4af7-af85-8f408a4654e9" width="35" height="35"> Show Your Support</h2>
+</div>
 
-3. **Configure Environment Variables**
+If you find this project interesting and inspiring, please consider showing your support by starring it on GitHub! Your star goes a long way in helping me reach more developers and encourages me to keep enhancing the project.
 
-   ```bash
-   cp .env.sample .env.local
-   ```
+• Feel free to get in touch with me for any further queries or support, happy to help :)
 
-4. **Launch the Development Server**
+<hr>
 
-   ```bash
-   npm run dev
-   ```
+## 📞Contact 
```

Comment:
space between the things, but we can remove this completely my x and linkendin not required here